cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -74,6 +75,7 @@ def test_vm_reconfig_add_remove_hw_cold(
         message="confirm that previously-added {} was removed".format(change_type))
 
 
+@pytest.mark.rhv1
```

Comment:
Also don't think that this is sanity

Reply:
Add disk is a cfme-5.8 new feature, added to cover basic functionality for RHV. It was included in the manual sanity.
Now it is no longer ran there (having it covered in automation). Only the rest api test is included.